test/integration/tests/path_folder_test.rb

```diff
@@ -76,7 +72,11 @@
 
     it 'has selinux label handling' do
       res = Test.selinux_label(backend, file.path)
-      file.selinux_label.must_equal(res)
+      if res.nil?
```

Comment:
Why do you expect that is a result from a method is nil, that this is okay?

Reply:
As noted above, you already returned `nil` here during the Docker tests, and I extended that to also return nil on Kubernetes tests. You weren't actually handling that in most of the tests so I fixed that while making the Kubernetes tests pass.